src/crypto/aes.rs

```diff
@@ -7,7 +7,7 @@ use crate::internal::{take_lock, IronOxideErr};
 use std::{convert::TryFrom, ops::DerefMut, sync::Mutex};
 
 //There is no way this can fail. Value is most definitely not less than one.
-const PBKDF2_ITERATIONS: NonZeroU32 = unsafe { NonZeroU32::new_unchecked(250000) };
+const PBKDF2_ITERATIONS: NonZeroU32 = unsafe { NonZeroU32::new_unchecked(250_000) };
```

Comment:
We'd discussed this formatting before and decided against it. I'm fine with it, however, if others are.

Reply:
I've talked to @coltfred about it in the past, and our consensus was to use it sparingly only when it makes sense. So here I think it improves readability, but numbers in gridiron are mostly incomprehensible anyway, so it would only make it more cluttered.